test/test.js

```diff
@@ -294,6 +294,45 @@ test('Remove mockjax definition by id', function() {
 	});
 });
 
+test('Remove mockjax definition by url', function() {
+	$.mockjax({
+		url: '/test',
+		contentType: 'text/plain',
+		responseText: 'test'
+	});
+
+	$.mockjax({
+		url: '*',
+		contentType: 'text/plain',
+		responseText: 'default'
+	});
+
+	stop();
+	$.ajax({
+		url: '/test',
+		success: function(text) {
+			equal(text, 'test', 'Test handler responded');
+		},
+		error: noErrorCallbackExpected,
+		complete: function() {
+			$.mockjax.clear('/test');
+
+			// Reissue the request expecting the default handler
+			$.ajax({
+				url: '/test',
+				success: function(text) {
+					equal(text, 'default', 'Default handler responded');
+				},
+				error: noErrorCallbackExpected,
+				complete: function(xhr) {
+					equal(xhr.responseText, 'default', 'Default handler responded');
+					start();
+				}
+			});
+		}
+	});
+});
+
```

Comment:
Can you add a few extra tests?

- [ ] attempt to 'clear' a non-existent (but similar) url match and ensure that the mock still handles the request
- [ ] match a regular expression and clear it
- [ ] no default handler and ensure that we get an error callback
- [ ] mock a subpath as well ('/test/foo') and make sure that once we remove the `/test` handler that the subpath still works